coderd/database/queries/activitybump.sql

```diff
@@ -0,0 +1,42 @@
+-- We bump by the original TTL to prevent counter-intuitive behavior
+-- as the TTL wraps. For example, if I set the TTL to 12 hours, sign off
+-- work at midnight, come back at 10am, I would want another full day
+-- of uptime. In the prior implementation, the workspace would enter
+-- a state of always expiring 1 hour in the future.
```

Comment:
Thanks for documenting! ❤️

Reply:
These are just Ammar's words :-)

Reply:
Ah, then perhaps we should remove "In the prior implementation", this had be confused that we now changed behavior 😅